magda-sleuther-broken-link/src/sleuther.ts

```diff
@@ -0,0 +1,441 @@
+import Registry from "@magda/typescript-common/dist/Registry";
+import { Record } from "@magda/typescript-common/dist/generated/registry/api";
+import unionToThrowable from "@magda/typescript-common/dist/util/union-to-throwable";
```

Comment:
But also I'd generally suggest exporting one thing per file, for the same reason.  That api.ts is an exception because it's generated.